custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json

```diff
@@ -366,6 +447,30 @@
                 "use_seasonal_model": "True"
             }
         },
+        {
+            "type": "SEPARATOR",
+            "visibilityCondition": "model.forecasting_style == 'customize_algorithms'"
+        },
+        {
+            "name": "stl_model_activated",
+            "label": "- Seasonal Trend",
+            "description": "Based on the statsmodels package STL model with ETS as forecast model.",
+            "type": "BOOLEAN",
+            "defaultValue": false,
+            "mandatory": true,
+            "visibilityCondition": "['choose_algorithms', 'customize_algorithms'].includes(model.forecasting_style)"
+        },
+        {
+            "name": "stl_model_kwargs",
+            "type": "MAP",
+            "label": "  ↳ Parameters",
+            "description": "Optional keyword arguments from https://www.statsmodels.org/stable/generated/statsmodels.tsa.forecasting.stl.STLForecast.html",
+            "visibilityCondition": "model.forecasting_style == 'customize_algorithms' && model.stl_model_activated",
+            "mandatory": false,
+            "defaultValue": {
```

Comment:
The default value shouldn't have a seasonal argument, since we use it from the top level Seasonality parameter.
I would suggest instead to do like for DeepAR, a model kwargs with the ETSmodel as default value.

Reply:
`seasonal` here is a different parameter from season_length.
which default model_kwargs to use ?
all params are optional (https://www.statsmodels.org/stable/generated/statsmodels.tsa.exponential_smoothing.ets.ETSModel.html?highlight=etsmodel#statsmodels.tsa.exponential_smoothing.ets.ETSModel)

Reply:
Okay, my bad. The term "seasonal" may lead to some confusion IMO. I would rather use model = ETSModel as a default kwarg in the UX.

Reply:
default kwargs is model=ETSModel
5b19a60